test/general/build_targets/package_test.dart

```diff
@@ -93,6 +95,7 @@ void main() {
           '-o',
           '${outputDir.path}/tpk/',
           '${projectDir.path}/tizen',
+          '/p:DefineConstants=${deviceProfile.toUpperCase()}_PROFILE',
```

Comment:
Using inlined values makes the test more robust because you can check if the value is parsed and passed to the builder correctly as well.
```suggestion
          '/p:DefineConstants=COMMON_PROFILE',
```

Reply:
done